### PR TITLE
Fixed UI test failures

### PR DESCRIPTION
**Pull request by sghai**
- Fixed `test_remove_env` by asserting the removed entity via search instead of success notification
- Fixed `test_remove_repo by updating locator for 'Remove_repository' button
- Fixed `test_delete_positive_1` config group test
- Fixed `test_positive_upate_2` by updating locator for `repo.fetch_gpgkey`
- Fixed `test_negative_create_usergroup_{2,3}` by updating `setUpClass`
- Fixed `test_update_os_template` by updating org association with config template

### CODE REVIEW

**Comment by sghai** [comment]
```@Test: Delete an environment ... ok

----------------------------------------------------------------------
Ran 1 test in 61.590s

OK
```

**Comment by sghai** [comment]
```
@Test: Create content repository and remove it ... ok

----------------------------------------------------------------------
Ran 1 test in 99.795s

OK
```

**Comment by sghai** [comment]
```
@Test: Delete selected config-groups ... ok

----------------------------------------------------------------------
Ran 1 test in 71.849s

OK
```

**Comment by sghai** [comment]
```
@Test: Update content repository with new gpg-key ... ok

----------------------------------------------------------------------
Ran 1 test in 107.373s

OK
```

**Comment by sghai** [comment]
```
@Test: Create a new UserGroup with blank and whitespace in name ... ok

----------------------------------------------------------------------
Ran 1 test in 60.732s

OK
```

```
@Test: Create a new UserGroup with same name ... ok

----------------------------------------------------------------------
Ran 1 test in 66.814s

OK
```

**Comment by sghai** [comment]
```
@Test: Update provisioning template ... ok

----------------------------------------------------------------------
Ran 1 test in 59.478s

OK
```

**Comment by elyezer** [comment]
ACK

File: tests/foreman/ui/test_operatingsys.py [hunk 1]
```diff
@@ -395,12 +395,14 @@ def test_update_os_template(self):
         """
         os_name = gen_string("alpha", 4)
         template_name = gen_string("alpha", 4)
-        os_attrs = entities.OperatingSystem(name=os_name).create()
+        os_attrs = entities.OperatingSystem(name=os_name).create_json()
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_operatingsys.py]: Thank you.

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**